### ISSUE TITLE
[Accessibility][Name, Role, Value - Bot Responses]: Aria-hidden elements contain focusable elements.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2025]
**User Experience:** 
This issue should be resolved for good coding practice as Aria-hidden elements contain focusable elements.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home.
2. Run K4W tool on the page and observe the issue in Left navigation pane.

**Actual Result:**
ARIA hidden elements contains focusable elements. Screen reader does not announce the selected/non-selected state of all controls in left navigation as screen reader only read like: Bot responses button.

**Expected Result:**
Ensures ARIA hidden elements do not contains focusable elements. Screen reader should read the selected/non-selected state of all controls in left navigation like: Bot responses button selected.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Attachment:**
[MAS4.1.2_Left navigation.zip](https://github.com/microsoft/BotFramework-Composer/files/4229299/MAS4.1.2_Left.navigation.zip)
[MAS4.1.2_Bot Responses _K4W.zip](https://github.com/microsoft/BotFramework-Composer/files/4229300/MAS4.1.2_Bot.Responses._K4W.zip)
[MAS4.1.2_Bot Responses_Left Navigation_K4W.zip](https://github.com/microsoft/BotFramework-Composer/files/4229301/MAS4.1.2_Bot.Responses_Left.Navigation_K4W.zip)
[MAS4.1.2_Bot Responses_Left Navigation_Screen reader.zip](https://github.com/microsoft/BotFramework-Composer/files/4229302/MAS4.1.2_Bot.Responses_Left.Navigation_Screen.reader.zip)

**Comment by corinagum** [comment]
![MAS4 1 2_Left navigation](https://user-images.githubusercontent.com/14900841/75414091-3399fa80-58dc-11ea-9137-94702a40ff54.png)

![MAS4 1 2_Bot Responses _K4W](https://user-images.githubusercontent.com/14900841/75414104-3dbbf900-58dc-11ea-9ffe-84ef2bca613b.png)

![MAS4 1 2_Bot Responses_Left Navigation_K4W](https://user-images.githubusercontent.com/14900841/75414114-48768e00-58dc-11ea-965e-f2383b3d6e14.png)

Only last .zip contains a video file. The first three contain .pngs, pasted above.

**Comment by beyackle** [comment]
This one is interesting. The elements inside the links are still focusable, even though they shouldn't be (the SR seems to think there's a 2xN grid here instead of a column; going down the "left" cells reads them as links, and going down the "right" cells reads them as buttons). Trying to just aria-hide the Buttons isn't working, either.

**Comment by ashish315** [comment]
@cwhitten As we checked the issue still repro at About button and Bot responses page.

**Comment by beyackle** [comment]
I'm not seeing any issue with the About button myself, but I will address the buttons on the Bot Responses page.

**Comment by beyackle** [comment]
PR coming out shortly.

![image](https://user-images.githubusercontent.com/61990921/78917525-d0a39380-7a43-11ea-9030-c21c58415d15.png)

**Comment by ashish315** [comment]
As checked the issue is fixed now. So, HCL is closing the issue.
